pkg/securitypolicy/securitypolicy_test.go

```diff
@@ -840,7 +840,7 @@ func setupContainerWithOverlay(gc *generatedContainers, valid bool) (tc *testCon
 }
 
 func generateContainers(r *rand.Rand, upTo int32) *generatedContainers {
-	var containers []securityPolicyContainer
+	var containers []*securityPolicyContainer
```

Comment:
unrelated to this PR:
can we switch to [fuzzing](https://go.dev/doc/fuzz/) so the seed can be specified from the command line and add the ability to replicate failing test conditions?

Reply:
thanks! I'll have a look later.